src/frontend/src/views/Upload/components/Metadata/components/ImportFile/prepMetadataTemplate.ts

```diff
@@ -3,17 +3,20 @@
  */
 import { METADATA_KEYS_TO_HEADERS } from "../../../common/constants";
 
+// Should change below whenever there are material changes to TSV download
+export const TEMPLATE_UPDATED_DATE = "2022-02-03"; // YYYY-MM-DD
```

Comment:
2022-02-03 is the likely date of this being deployed to Prod, assuming it gets merged in before then. If it isn't merged before the next Prod push, I'll update this to whatever the next Prod push date will be.